diff -Nur orig/src/compiler/nir/nir.h mod/src/compiler/nir/nir.h
--- orig/src/compiler/nir/nir.h	2019-12-04 23:10:12.000000000 +0100
+++ mod/src/compiler/nir/nir.h	2021-01-13 12:53:09.390209790 +0100
@@ -42,9 +42,9 @@
 #include "compiler/shader_info.h"
 #include <stdio.h>
 
-#ifndef NDEBUG
+#ifdef DEBUG
 #include "util/debug.h"
-#endif /* NDEBUG */
+#endif /* DEBUG */
 
 #include "nir_opcodes.h"
 
@@ -3222,7 +3222,7 @@
 
 void nir_shader_serialize_deserialize(nir_shader *s);
 
-#ifndef NDEBUG
+#ifdef DEBUG
 void nir_validate_shader(nir_shader *shader, const char *when);
 void nir_metadata_set_validation_flag(nir_shader *shader);
 void nir_metadata_check_validation_flag(nir_shader *shader);
@@ -3281,7 +3281,7 @@
 static inline bool should_clone_nir(void) { return false; }
 static inline bool should_serialize_deserialize_nir(void) { return false; }
 static inline bool should_print_nir(void) { return false; }
-#endif /* NDEBUG */
+#endif /* DEBUG */
 
 #define _PASS(pass, nir, do_pass) do {                               \
    if (should_skip_nir(#pass)) {                                     \
diff -Nur orig/src/compiler/nir/nir_metadata.c mod/src/compiler/nir/nir_metadata.c
--- orig/src/compiler/nir/nir_metadata.c	2019-12-04 23:10:12.000000000 +0100
+++ mod/src/compiler/nir/nir_metadata.c	2021-01-13 12:54:00.554222976 +0100
@@ -59,7 +59,7 @@
    impl->valid_metadata &= preserved;
 }
 
-#ifndef NDEBUG
+#ifdef DEBUG
 /**
  * Make sure passes properly invalidate metadata (part 1).
  *
diff -Nur orig/src/compiler/nir/nir_validate.c mod/src/compiler/nir/nir_validate.c
--- orig/src/compiler/nir/nir_validate.c	2019-12-04 23:10:12.000000000 +0100
+++ mod/src/compiler/nir/nir_validate.c	2021-01-13 12:54:37.402231146 +0100
@@ -36,7 +36,7 @@
 /* Since this file is just a pile of asserts, don't bother compiling it if
  * we're not building a debug build.
  */
-#ifndef NDEBUG
+#ifdef DEBUG
 
 /*
  * Per-register validation state.
diff -Nur orig/src/mesa/main/mtypes.h mod/src/mesa/main/mtypes.h
--- orig/src/mesa/main/mtypes.h	2019-12-04 23:10:14.000000000 +0100
+++ mod/src/mesa/main/mtypes.h	2021-01-13 13:30:58.158768488 +0100
@@ -5189,7 +5189,7 @@
    unsigned nr_device_memory_evictions; /**< # of evictions (monotonic counter) */
 };
 
-#ifndef NDEBUG
+#ifdef DEBUG
 extern int MESA_VERBOSE;
 extern int MESA_DEBUG_FLAGS;
 #else
