diff -Naur orig/CMakeLists.txt patched/CMakeLists.txt
--- orig/CMakeLists.txt	2020-12-19 02:43:12.000000000 +0100
+++ patched/CMakeLists.txt	2021-02-22 20:35:02.302797927 +0100
@@ -502,6 +502,7 @@
 	pkg_check_modules( CAIRO REQUIRED cairo>=1.10 )
 	pkg_check_modules( PANGOCAIRO REQUIRED pangocairo>=1.22.0 )
 	pkg_check_modules( FONTCONFIG REQUIRED fontconfig>=2.4.1 )
+	pkg_check_modules( GLIB REQUIRED gobject-2.0)
 	pkg_check_modules( SYSTEMD systemd )
 endif(ENABLE_GAME OR ENABLE_TESTS)
 
diff -Naur orig/src/CMakeLists.txt patched/src/CMakeLists.txt
--- orig/src/CMakeLists.txt	2020-12-19 02:43:12.000000000 +0100
+++ patched/src/CMakeLists.txt	2021-02-22 20:34:00.406547741 +0100
@@ -50,6 +50,7 @@
 	${Boost_LIBRARY_DIRS}
 	${PANGOCAIRO_LIBRARY_DIRS}
 	${FONTCONFIG_LIBRARY_DIRS}
+	${GLIB_LIBRARY_DIRS}
 	${LIBDBUS_LIBRARY_DIRS}
 )
 
@@ -80,6 +81,7 @@
 	${sdl_ttf-lib}
 	${PANGOCAIRO_LIBRARIES}
 	${FONTCONFIG_LIBRARIES}
+	${GLIB_LIBRARIES}
 	${LIBDBUS_LIBRARIES}
 	${VORBISFILE_LIBRARIES}
 )
