diff -Nur orig/CMakeLists.txt mod/CMakeLists.txt
--- orig/CMakeLists.txt	2022-05-29 17:04:52.000000000 +0200
+++ mod/CMakeLists.txt	2022-06-02 17:03:38.388601523 +0200
@@ -242,8 +242,8 @@
 			set(CMAKE_CXX_FLAGS_RELEASE "-Og")
 			set(CMAKE_C_FLAGS_RELEASE "-Og")
 		else()
-			set(CMAKE_CXX_FLAGS_RELEASE "-O3")
-			set(CMAKE_C_FLAGS_RELEASE "-O3")
+			set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+			set(CMAKE_C_FLAGS_RELEASE "-O2")
 		endif()
 	else()
 		set(CMAKE_CXX_FLAGS_RELEASE "${OPT}")
@@ -501,6 +501,7 @@
 	pkg_check_modules(FONTCONFIG REQUIRED fontconfig>=2.4.1)
 	pkg_check_modules(PANGO REQUIRED pango>=1.22.0)
 	pkg_check_modules(LIBREADLINE readline)
+	pkg_check_modules( GLIB REQUIRED gobject-2.0)
 endif()
 
 if(ENABLE_TESTS)
diff -Nur orig/src/CMakeLists.txt mod/src/CMakeLists.txt
--- orig/src/CMakeLists.txt	2022-05-29 17:04:52.000000000 +0200
+++ mod/src/CMakeLists.txt	2022-06-02 17:04:59.196853043 +0200
@@ -59,6 +59,7 @@
 if(NOT MSVC)
 	set(game-external-libs
 		${game-external-libs}
+        ${GLIB_LIBRARIES}
 		${SDL2_LIBRARIES}
 		${SDL2IMAGE_LIBRARIES}
 		${SDL2MIXER_LIBRARIES}
