diff -Nur orig/CMakeLists.txt mod/CMakeLists.txt
--- orig/CMakeLists.txt	2021-10-24 02:23:51.000000000 +0200
+++ mod/CMakeLists.txt	2021-10-29 19:49:08.938793138 +0200
@@ -242,8 +242,8 @@
 			set(CMAKE_CXX_FLAGS_RELEASE "-Og")
 			set(CMAKE_C_FLAGS_RELEASE "-Og")
 		else()
-			set(CMAKE_CXX_FLAGS_RELEASE "-O3")
-			set(CMAKE_C_FLAGS_RELEASE "-O3")
+			set(CMAKE_CXX_FLAGS_RELEASE "-O2")
+			set(CMAKE_C_FLAGS_RELEASE "-O2")
 		endif()
 	else()
 		set(CMAKE_CXX_FLAGS_RELEASE "${OPT}")
@@ -486,6 +486,7 @@
 	pkg_check_modules(PANGOCAIRO REQUIRED pangocairo>=1.22.0)
 # TODO: can use find_package and Fontconfig::Fontconfig with cmake 3.14+
 	pkg_check_modules(FONTCONFIG REQUIRED fontconfig>=2.4.1)
+	pkg_check_modules( GLIB REQUIRED gobject-2.0)
 endif()
 
 if(ENABLE_TESTS)
diff -Nur orig/src/CMakeLists.txt mod/src/CMakeLists.txt
--- orig/src/CMakeLists.txt	2021-10-24 02:23:51.000000000 +0200
+++ mod/src/CMakeLists.txt	2021-10-29 19:45:25.745914210 +0200
@@ -34,6 +34,7 @@
 link_directories(
 	${PANGOCAIRO_LIBRARY_DIRS}
 	${FONTCONFIG_LIBRARY_DIRS}
+	${GLIB_LIBRARY_DIRS}
 	${LIBDBUS_LIBRARY_DIRS}
 )
 
@@ -53,6 +54,7 @@
 	${SDL2_MIXER_LIBRARY}
 	${PANGOCAIRO_LIBRARIES}
 	${FONTCONFIG_LIBRARIES}
+	${GLIB_LIBRARIES}
 	${LIBDBUS_LIBRARIES}
 	${VORBISFILE_LIBRARIES}
 )
