diff -Nur orig/setup/build_environment.py mod/setup/build_environment.py
--- orig/setup/build_environment.py	2020-02-07 05:49:31.000000000 +0100
+++ mod/setup/build_environment.py	2020-02-12 18:07:36.426108124 +0100
@@ -110,7 +110,7 @@
             if not os.path.exists(q):
                 q = os.path.join(sys.prefix, 'share', 'sip')
     q = os.environ.get('SIP_DIR', q)
-    for x in ('', 'Py2-PyQt5', 'PyQt5', 'sip/PyQt5'):
+    for x in ('', 'Py3-PyQt5', 'PyQt5', 'sip/PyQt5'):
         base = os.path.join(q, x)
         if os.path.exists(os.path.join(base, 'QtWidgets')):
             return base
