diff -Nur orig/configure.ac mod/configure.ac
--- orig/configure.ac	2019-05-17 06:54:08.000000000 +0200
+++ mod/configure.ac	2021-03-17 19:56:06.469228539 +0100
@@ -221,7 +221,6 @@
 dnl Documentation
 dnl ###########################################################################
 
-GTK_DOC_CHECK
 YELP_HELP_INIT
 
 dnl ###########################################################################
@@ -240,7 +239,6 @@
 Makefile
 common/Makefile
 data/Makefile
-doc/Makefile
 help/Makefile
 liblightdm-gobject/liblightdm-gobject-1.pc
 liblightdm-gobject/Makefile
diff -Nur orig/Makefile.am mod/Makefile.am
--- orig/Makefile.am	2019-05-17 06:54:08.000000000 +0200
+++ mod/Makefile.am	2021-03-17 20:09:31.248408291 +0100
@@ -3,9 +3,9 @@
 if COMPILE_TESTS
 SUBDIRS += tests
 endif
-SUBDIRS += data po src doc help
+SUBDIRS += data po src help
 
-DISTCHECK_CONFIGURE_FLAGS = --enable-introspection --enable-gtk-doc
+DISTCHECK_CONFIGURE_FLAGS = --enable-introspection
 
 EXTRA_DIST = \
 	autogen.sh \
