diff -Naur orig/data/alltray.desktop.in mod/data/alltray.desktop.in
--- orig/data/alltray.desktop.in	2011-07-21 07:59:43.000000000 +0200
+++ mod/data/alltray.desktop.in	2016-04-12 16:48:27.000000000 +0200
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=AllTray @PACKAGE_VERSION@
-GenericName=AllTray @PACKAGE_VERSION@
+Name=AllTray
+GenericName=AllTray
 Comment=Dock everything to the system tray/notification area.
 Exec=alltray -a
-Icon=alltray.png
+Icon=alltray
 Terminal=false
 Type=Application
 Categories=Application;Utility;
