diff -Nur orig/GPG mod/GPG
--- orig/GPG	2020-03-22 00:53:55.000000000 +0100
+++ mod/GPG	2022-06-18 13:11:10.340091094 +0200
@@ -1,5 +1,8 @@
 gpg-s tároló használata UBK kulcssal
 
+--------------------------------------------------------------------------------
+Régebbi (< 2.3.15) apt esetén:
+
 root-ként:
 gpg --keyserver hkp://keys.gnupg.net --recv-keys 12A3276B
 gpg --import ubk-secret.asc
@@ -19,3 +22,31 @@
 gpg --import utvonal/sajat.asc
 gpg --import utvonal/sajat-privat.asc
 apt-key add utvonal/sajat.asc
+
+--------------------------------------------------------------------------------
+apt 2.3.15-től:
+Ha létezik az /etc/apt/trusted.gpg, töröljük.
+
+root-ként:
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 12A3276B
+gpg --import ubk-secret.asc
+gpg --export 12A3276B > ubk.gpg
+mv ubk.gpg /etc/apt/trusted.gpg.d/ubk.gpg
+  VAGY
+wget http://ubk.hu/gpg/ubk.gpg
+mv ubk.gpg /etc/apt/trusted.gpg.d/ubk.gpg
+
+saját új kulccsal:
+root-ként:
+gpg --gen-key
+gpg --export kulcsazonosító > utvonal/sajat.gpg
+mv utvonal/sajat.gpg /etc/apt/trusted.gpg.d/sajat.gpg
+
+saját meglévő kulccsal:
+userként:
+gpg --export kulcsazonosító > utvonal/sajat.gpg
+gpg --export-secret-key -a kulcsazonosító > utvonal/sajat-privat.asc
+root-ként:
+gpg --import utvonal/sajat.gpg
+gpg --import utvonal/sajat-privat.asc
+mv utvonal/sajat.gpg /etc/apt/trusted.gpg.d/sajat.gpg
