diff -Nur orig/src/uhubuild-u2d.sh mod/src/uhubuild-u2d.sh
--- orig/src/uhubuild-u2d.sh	2020-03-22 00:53:55.000000000 +0100
+++ mod/src/uhubuild-u2d.sh	2021-03-28 20:48:28.000000000 +0200
@@ -133,16 +133,22 @@
 sflist $1 $2/$subdir | parsever | sort -V | tail -n 1
 }
 
-# script for gnome projects
+# script for gnome and gnome-core projects
 function u2dgnome {
 	project="$1"
 	unstable="$2"
 	reg="02468";
+if [ ! "$(echo $unstable | grep -E '^3')" ];then
 	if [ -n "${unstable:-}" ]; then
 		reg="0-9"
 	fi
 	ver="$(urllist "https://download.gnome.org/sources/$project/" | grep -E '^[0-9]+\.[0-9]*['$reg'](\.[0-9.])?$' | sort -V | tail -n 1)"
-	u2d "https://download.gnome.org/sources/$project/$ver"
+	u2d "https://download.gnome.org/sources/$project/$ver/"
+	exit
+	else
+	url=https://download.gnome.org/core/$unstable/
+	u2d $url/$(urllist $url/ | grep -E '^([0-9.]+)' | sort -V | tail -n 1)/sources/ $project-
+fi
 }
 
 # script for mate projects
