diff -Nur orig/Makefile mod/Makefile
--- orig/Makefile	2021-05-05 18:36:58.000000000 +0200
+++ mod/Makefile	2021-05-23 18:26:56.629191733 +0200
@@ -41,7 +41,7 @@
 endif
 
 _CFLAGS += $(shell pkg-config --cflags alsa libva libavcodec libswscale libswresample x11 x11-xcb xcb xcb-icccm xcb-screensaver xcb-dpms)
-LIBS += -lrt $(shell pkg-config --libs alsa libva libavcodec libswscale libswresample x11 x11-xcb xcb xcb-icccm xcb-screensaver xcb-dpms)
+LIBS += -lrt $(shell pkg-config --libs alsa libva-x11 libavcodec libswscale libswresample x11 x11-xcb xcb xcb-icccm xcb-screensaver xcb-dpms)
 
 ### The version number of VDR's plugin API:
 
