diff -Nur orig/server/tools.c mod/server/tools.c
--- orig/server/tools.c	2010-07-09 15:19:33.000000000 +0200
+++ mod/server/tools.c	2022-03-23 19:08:32.015994310 +0100
@@ -261,8 +261,6 @@
         if (user_to_switch != NULL) {
                 struct passwd* user = getpwnam(user_to_switch);
                 if (user) {
-                        setgid(user->pw_gid);
-                        setuid(user->pw_uid);
                 } else {
                         l2(OUTPUT_TYPE_ERROR, "Cannot switch user to %s: %s", user_to_switch, strerror(errno));
                 }
