diff -Nur orig/wpa_supplicant/Makefile mod/wpa_supplicant/Makefile
--- orig/wpa_supplicant/Makefile	2022-03-20 18:02:26.000000000 +0100
+++ mod/wpa_supplicant/Makefile	2022-03-20 18:07:04.164427932 +0100
@@ -30,9 +30,9 @@
 endif
 endif
 
-export LIBDIR ?= /usr/local/lib
-export INCDIR ?= /usr/local/include
-export BINDIR ?= /usr/local/sbin
+export LIBDIR ?= /usr/lib
+export INCDIR ?= /usr/include
+export BINDIR ?= /usr/sbin
 PKG_CONFIG ?= pkg-config
 
 CFLAGS += $(EXTRA_CFLAGS)
