diff -Naur orig/gmic-qt/CMakeLists.txt patched/gmic-qt/CMakeLists.txt
--- orig/gmic-qt/CMakeLists.txt	2020-02-10 16:35:00.000000000 +0100
+++ patched/gmic-qt/CMakeLists.txt	2020-02-17 23:58:42.623047456 +0100
@@ -482,7 +482,7 @@
     ${gmic_translation_files}
   )
 
-install(FILES ${gmic_qt_QM} DESTINATION ${CMAKE_SOURCE_DIR}/translations)
+# install(FILES ${gmic_qt_QM} DESTINATION ${CMAKE_SOURCE_DIR}/translations)
 
 set(gmic_qt_QRC
     gmic_qt.qrc
