diff -Nur orig/panels/color/cinnamon-color-panel.desktop.in.in mod/panels/color/cinnamon-color-panel.desktop.in.in
--- orig/panels/color/cinnamon-color-panel.desktop.in.in	2021-11-15 14:01:48.000000000 +0100
+++ mod/panels/color/cinnamon-color-panel.desktop.in.in	2021-11-27 19:43:02.701547488 +0100
@@ -4,8 +4,8 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GTK;Settings;X-Cinnamon-Settings-Panel;HardwareSettings
-OnlyShowIn=X-Cinnamon;
+Categories=GTK;Settings;X-Cinnamon-Settings-Panel;HardwareSettings;
+OnlyShowIn=Cinnamon;
 X-Cinnamon-Settings-Panel=color
 # Translators: those are keywords for the color control-center panel
 _Keywords=Color;ICC;Profile;Calibrate;Printer;Display;
diff -Nur orig/panels/display/cinnamon-display-panel.desktop.in.in mod/panels/display/cinnamon-display-panel.desktop.in.in
--- orig/panels/display/cinnamon-display-panel.desktop.in.in	2021-11-15 14:01:48.000000000 +0100
+++ mod/panels/display/cinnamon-display-panel.desktop.in.in	2021-11-27 19:43:19.897786176 +0100
@@ -5,7 +5,7 @@
 Type=Application
 StartupNotify=true
 Categories=GTK;Settings;HardwareSettings;X-Cinnamon-Settings-Panel;
-OnlyShowIn=X-Cinnamon;
+OnlyShowIn=Cinnamon;
 X-Cinnamon-Settings-Panel=display
 # Translators: those are keywords for the display control-center panel
 _Keywords=Panel;Projector;xrandr;Screen;Resolution;Refresh;
diff -Nur orig/panels/network/cinnamon-network-panel.desktop.in.in mod/panels/network/cinnamon-network-panel.desktop.in.in
--- orig/panels/network/cinnamon-network-panel.desktop.in.in	2021-11-15 14:01:48.000000000 +0100
+++ mod/panels/network/cinnamon-network-panel.desktop.in.in	2021-11-27 19:43:36.366010636 +0100
@@ -5,7 +5,7 @@
 Type=Application
 StartupNotify=true
 Categories=GTK;Settings;HardwareSettings;X-Cinnamon-Settings-Panel;
-OnlyShowIn=X-Cinnamon;
+OnlyShowIn=Cinnamon;
 X-Cinnamon-Settings-Panel=network
 # Translators: those are keywords for the network control-center panel
 _Keywords=Network;Wireless;IP;LAN;Proxy;Internet;WiFi;
diff -Nur orig/panels/wacom/cinnamon-wacom-panel.desktop.in.in mod/panels/wacom/cinnamon-wacom-panel.desktop.in.in
--- orig/panels/wacom/cinnamon-wacom-panel.desktop.in.in	2021-11-15 14:01:48.000000000 +0100
+++ mod/panels/wacom/cinnamon-wacom-panel.desktop.in.in	2021-11-27 19:43:51.938219243 +0100
@@ -6,7 +6,7 @@
 StartupNotify=true
 Categories=GNOME;GTK;Settings;HardwareSettings;X-Cinnamon-Settings-Panel;
 X-Cinnamon-Settings-Panel=wacom
-OnlyShowIn=X-Cinnamon;
+OnlyShowIn=Cinnamon;
 # Translators: those are keywords for the wacom tablet control-center panel
 _Keywords=Tablet;Wacom;Stylus;Eraser;Mouse;
 Name=Graphics Tablet
