diff -Naur orig/guicast/bccmdl.py patched/guicast/bccmdl.py
--- orig/guicast/bccmdl.py	2019-07-31 00:01:49.000000000 +0200
+++ patched/guicast/bccmdl.py	2020-03-31 18:06:52.415833568 +0200
@@ -1,4 +1,4 @@
-#!/bin/python
+#!/bin/python2
 
 # Retain python2 compatibility
 from __future__ import print_function
diff -Naur orig/guicast/Makefile patched/guicast/Makefile
--- orig/guicast/Makefile	2019-06-15 20:47:51.000000000 +0200
+++ patched/guicast/Makefile	2020-03-31 18:06:43.803714181 +0200
@@ -95,7 +95,7 @@
 all: $(OUTPUT) $(UTILS)
 
 $(BCXFER):	bccmdl.py bcxfer.C bcxfer.h
-	python < ./bccmdl.py
+	python2 < ./bccmdl.py
 	+$(MAKE) -C xfer
 
 $(OUTPUT): $(OBJS) $(BCXFER)
