diff -Nur orig/wireshark.desktop mod/wireshark.desktop
--- orig/wireshark.desktop	2021-07-14 18:47:58.000000000 +0200
+++ mod/wireshark.desktop	2021-11-14 10:40:11.502437646 +0100
@@ -103,7 +103,7 @@
 Comment[uk]=Аналізатор мережевого трафіку
 Icon=wireshark
 TryExec=wireshark
-Exec=wireshark %f
+Exec=gksu wireshark
 Terminal=false
 MimeType=application/vnd.tcpdump.pcap;application/x-pcapng;application/x-snoop;application/x-iptrace;application/x-lanalyzer;application/x-nettl;application/x-radcom;application/x-etherpeek;application/x-visualnetworks;application/x-netinstobserver;application/x-5view;application/x-tektronix-rf5;application/x-micropross-mplog;application/x-apple-packetlogger;application/x-endace-erf;application/ipfix;application/x-ixia-vwr;
 # Category entry according to:
