diff -Nur orig/data/supertuxkart.desktop mod/data/supertuxkart.desktop
--- orig/data/supertuxkart.desktop	2020-08-28 10:16:23.000000000 +0200
+++ mod/data/supertuxkart.desktop	2021-02-07 19:16:41.501184742 +0100
@@ -6,6 +6,7 @@
 GenericName[de]=Ein Kart-Rennspiel
 GenericName[fr]=Un jeu de karting
 GenericName[gl]=Xogo de carreiras con karts
+GenericName[hu]=Gokart játék
 GenericName[pl]=Wyścigi gokartów
 GenericName[ro]=Un joc de curse cu carturi
 Exec=supertuxkart
@@ -14,4 +15,3 @@
 Type=Application
 Categories=Game;ArcadeGame;
 Keywords=tux;game;race;
-PrefersNonDefaultGPU=true
