Index: Cinnamon-3.6.6/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py
===================================================================
--- Cinnamon-3.6.6.orig/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py
+++ Cinnamon-3.6.6/files/usr/share/cinnamon/cinnamon-settings-users/cinnamon-settings-users.py
@@ -53,10 +53,9 @@
             print(detail)
 
     def _on_entry_changed(self, entry):
-        name = entry.get_text()
-        if " " in name or name.lower() != name:
+        if " " in entry.get_text():
             entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, "dialog-warning-symbolic")
-            entry.set_icon_tooltip_text(Gtk.EntryIconPosition.SECONDARY, _("The group name cannot contain upper-case or space characters"))
+            entry.set_icon_tooltip_text(Gtk.EntryIconPosition.SECONDARY, _("The group name cannot contain space characters"))
             self.set_response_sensitive(Gtk.ResponseType.OK, False)
         else:
             entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, None)
@@ -405,9 +405,9 @@
         fullname = self.realname_entry.get_text()
         username = self.username_entry.get_text()
         valid = True
-        if re.search('[^a-z0-9_.-]', username):
+        if " " in username:
             self.username_entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, "dialog-warning-symbolic")
-            self.username_entry.set_icon_tooltip_text(Gtk.EntryIconPosition.SECONDARY, _("Invalid username"))
+            self.username_entry.set_icon_tooltip_text(Gtk.EntryIconPosition.SECONDARY, _("The username cannot contain space characters"))
             valid = False
         elif self.user_exists(username):
             self.username_entry.set_icon_from_icon_name(Gtk.EntryIconPosition.SECONDARY, "dialog-warning-symbolic")

