diff -Nur orig/imageloader.c mod/imageloader.c
--- orig/imageloader.c	2014-05-09 13:05:44.000000000 +0200
+++ mod/imageloader.c	2021-01-12 10:42:41.125439252 +0100
@@ -11,7 +11,6 @@
 using namespace Magick;
 
 cImageLoader::cImageLoader() {
-    InitializeMagick(NULL);
 }
 
 cImageLoader::~cImageLoader() {
@@ -183,4 +182,4 @@
     imgback.composite(imgblend, 0, 0, OverCompositeOp);
 
     buffer = imgback;
-}
\ Nincs új sor a fájl végén
+}
diff -Nur orig/imagemagickwrapper.c mod/imagemagickwrapper.c
--- orig/imagemagickwrapper.c	2014-05-09 13:05:44.000000000 +0200
+++ mod/imagemagickwrapper.c	2021-01-12 10:42:54.885282219 +0100
@@ -5,7 +5,6 @@
 #include "imagescaler.h"
 
 cImageMagickWrapper::cImageMagickWrapper() {
-    InitializeMagick(NULL);
 }
 
 cImageMagickWrapper::~cImageMagickWrapper() {
@@ -159,4 +158,4 @@
 
 void cImageMagickWrapper::CreateBackground(tColor back, tColor blend, int width, int height) {
     CreateGradient(back, blend, width, height, 0.8, 0.8);
-}
\ Nincs új sor a fájl végén
+}
diff -Nur orig/tvguide.c mod/tvguide.c
--- orig/tvguide.c	2014-05-11 08:57:57.000000000 +0200
+++ mod/tvguide.c	2021-01-12 10:47:23.698739954 +0100
@@ -20,7 +20,7 @@
 #include "config.h"
 #include "setup.h"
 #include "tvguideosd.h"
-
+#include <Magick++.h>
 
 #if defined(APIVERSNUM) && APIVERSNUM < 20000
 #error "VDR-2.0.0 API version or greater is required!"
@@ -93,6 +93,10 @@
   return true;
 }
 
+__attribute__((constructor)) static void init(void) {
+   Magick::InitializeMagick(NULL);
+}
+
 bool cPluginTvguide::Initialize(void) {
     tvguideConfig.SetDefaultPathes();
     tvguideConfig.LoadTheme();
