diff -Nur orig/Makefile mod/Makefile
--- orig/Makefile	2021-02-10 18:54:01.000000000 +0100
+++ mod/Makefile	2022-03-24 18:16:03.973318142 +0100
@@ -1,5 +1,5 @@
-LIBS= `pkg-config --libs glib-2.0 gtk+-2.0 libmenu-cache`
-CFLAGS+= -g -Wall `pkg-config --cflags glib-2.0 gtk+-2.0 libmenu-cache`
+LIBS= `pkg-config --libs glib-2.0 gtk+-3.0 libmenu-cache`
+CFLAGS+= -g -Wall `pkg-config --cflags glib-2.0 gtk+-3.0 libmenu-cache`
 CC=gcc
 #-DG_DISABLE_DEPRECATED
 
diff -Nur orig/src/menu.c mod/src/menu.c
--- orig/src/menu.c	2021-02-10 18:54:01.000000000 +0100
+++ mod/src/menu.c	2022-03-24 18:33:20.653086622 +0100
@@ -240,7 +240,6 @@
 
 #ifdef WITH_ICONS
 	gtk_init (&argc, &argv);
-	icon_theme = gtk_icon_theme_get_default ();
 #endif
 
 	if ((ob_context = configure (argc, argv)) == NULL)
diff -Nur orig/src/utils.c mod/src/utils.c
--- orig/src/utils.c	2021-02-10 18:54:01.000000000 +0100
+++ mod/src/utils.c	2022-03-24 18:21:30.818491653 +0100
@@ -169,8 +169,6 @@
 
 #if WITH_ICONS
 
-extern GtkIconTheme *icon_theme;
-
 /****f* utils/item_icon_path
  * OUTPUT
  *   return the path for the themed icon if item.
@@ -199,23 +197,19 @@
 		if (g_path_is_absolute (name))
 			return g_strdup (name);
 
-		/*  We remove the file extension as gtk_icon_theme_lookup_icon can't
-		 *  lookup a theme icon for, ie, 'geany.png'. It has to be 'geany'.
-		 */
-		tmp_name = strndup (name, strrchr (name, '.') - name);
 	#ifdef WITH_SVG
-		icon_info = gtk_icon_theme_lookup_icon (icon_theme, tmp_name, 16, GTK_ICON_LOOKUP_GENERIC_FALLBACK);
+		icon_info = gtk_icon_theme_lookup_icon (gtk_icon_theme_get_default(), name, 16, GTK_ICON_LOOKUP_GENERIC_FALLBACK);
 	#else
-		icon_info = gtk_icon_theme_lookup_icon (icon_theme, tmp_name, 16, GTK_ICON_LOOKUP_NO_SVG | GTK_ICON_LOOKUP_GENERIC_FALLBACK);
+		icon_info = gtk_icon_theme_lookup_icon (gtk_icon_theme_get_default(), name, 16, GTK_ICON_LOOKUP_NO_SVG | GTK_ICON_LOOKUP_GENERIC_FALLBACK);
 	#endif
 		g_free (tmp_name);
 	}
 
 	if (!icon_info) /* 2nd fallback */
-		icon_info = gtk_icon_theme_lookup_icon (icon_theme, "empty", 16, GTK_ICON_LOOKUP_NO_SVG);
+		icon_info = gtk_icon_theme_lookup_icon (gtk_icon_theme_get_default (), "empty", 16, GTK_ICON_LOOKUP_NO_SVG);
 
 	icon = g_strdup (gtk_icon_info_get_filename (icon_info));
-	gtk_icon_info_free (icon_info);
+	g_object_unref (icon_info);
 
 	return icon;
 }
